### PR TITLE
Admin Template view not found.

### PR DESCRIPTION
**Pull request by microposmp**
Fixes: #15140

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Templates/Controllers/TemplateController.cs [hunk 1]
```diff
@@ -115,7 +115,7 @@ public async Task<IActionResult> Index(ContentOptions options, PagerParameters p
                 new SelectListItem(S["Delete"], nameof(ContentsBulkAction.Remove)),
             ];
 
-            return View(model);
+            return View("Index", model);
```
Comment (MikeAlhayek) [inline src/OrchardCore.Modules/OrchardCore.Templates/Controllers/TemplateController.cs:118]: This isn't needed. By default View(model) calls a view with the same name as the action.

Comment (MikeAlhayek) [reply src/OrchardCore.Modules/OrchardCore.Templates/Controllers/TemplateController.cs:118]: I see. Then maybe we should add a document explaining the reason for needing in in the code. Also change please `"Index"` to `nameof(Index)`

Comment (microposmp) [reply src/OrchardCore.Modules/OrchardCore.Templates/Controllers/TemplateController.cs]: Added comment that describes the reason it's needed in this case.

File: src/OrchardCore.Modules/OrchardCore.Templates/Controllers/TemplateController.cs [hunk 2]
```diff
@@ -115,7 +115,8 @@ public async Task<IActionResult> Index(ContentOptions options, PagerParameters p
                 new SelectListItem(S["Delete"], nameof(ContentsBulkAction.Remove)),
             ];
 
-            return View(model);
+            // Both the Templates and Admin Templates use the same view (Index.cshtml) but with a different route.
```
Comment (MikeAlhayek) [inline src/OrchardCore.Modules/OrchardCore.Templates/Controllers/TemplateController.cs:118]: @microposmp 

Can you please update the comment to use this instead?

// Both the 'Template' and 'Admin' actions render the 'Index' template.
// To ensure we render the same view, we explicitly specify the name of the view that should be rendered.

**Review by MikeAlhayek — APPROVED** [review]

**Comment by Piedone** [comment]
Please change your PR review status to changes requested here @MikeAlhayek.

File: src/OrchardCore.Modules/OrchardCore.Templates/Controllers/TemplateController.cs (hunk 2 above)
Comment (MikeAlhayek) [inline src/OrchardCore.Modules/OrchardCore.Templates/Controllers/TemplateController.cs:118]: @microposmp

Can you please update the comment to use this instead?

// Both the 'Template' and 'Admin' actions render the 'Index' template.
// To ensure we render the same view, we explicitly specify the name of the view that should be rendered.

**Review by MikeAlhayek — CHANGES_REQUESTED** [review]
One last change and I'll merge it.

**Pull request merged**